pkg/build/api/types.go

```diff
@@ -42,8 +49,8 @@ type Build struct {
 // BuildSpec encapsulates all the inputs necessary to represent a build.
 type BuildSpec struct {
 	// ServiceAccount is the name of the ServiceAccount to use to run the pod
-	// created by this build.
-	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	// created by this build. The pod will be allowed to use secrets referenced
```

Comment:
I would have cosmetic changes in separate commits. But our single-commit approach and you comment about "unrelated changes" push us against making trivial improvements.

A separate pull request only for those changes was also not something that sounded like a great idea... how is @PI-Victor and we all in general supposed to pass in such kind of changes?